fission/src/systems/simulation/stimulus/WheelStimulus.ts

```diff
@@ -34,9 +34,9 @@ class WheelRotationStimulus extends EncoderStimulus {
         this._wheel = wheel;
     }
 
-    public Update(deltaT: number): void {
+    public Update(_: number): void {
         if (this._accum) {
-            this._wheelRotationAccum += this._wheel.GetAngularVelocity() * deltaT;
+            this._wheelRotationAccum += 0;
```

Comment:
Why'd you remove this?

Reply:
I wasn't sure what it did and removed it while debugging wheel behavior, but I just now added it back in my most recent commit.